doc/wiki/Installation:-Option-List.md

```diff
@@ -0,0 +1,147 @@
+|:---:|:---:|:---:|---|---|---|
+| `--flu_scheme` | `RTVD`, `MHM`, `MHM_RP`, `CTU`                             | `CTU`   | Hydro schemes. `RTVD`: relaxing TVD; `MHM`: MUSCL-Hancock; `MHM_RP`: VL scheme; `CTU`: corner transport upwind | `--mhd` only supports `MHM`, `MHM_RP`, and `CTU`; `--srhd` only supports `MHM` and `MHM_RP`; `--cosmic_ray` only supports `MHM_RP` | <a name="--flu_scheme"></a> `FLU_SCHEME` |
+| `--slope`      | `PLM`, `PPM`                                               | `PPM`   | Spatial reconstruction. `PLM`: piecewise linear; `PPM`: piecewise parabolic | Useless for `--flu_scheme=RTVD` | <a name="--slope"></a> `LR_SCHEME` |
+| `--flux`       | `EXACT`, `ROE`, `HLLE`, `HLLC`, `HLLD`                     | Depend  | Riemann solvers | Useless for `--flu_scheme=RTVD`; `EXACT` is experimental; pure hydrodynamics supports `EXACT/ROE/HLLE/HLLC`; `--mhd` supports `ROE/HLLE/HLLD`; `--srhd` and `--cosmic_ray` support `HLLE/HLLC` | <a name="--flux"></a> `RSOLVER` |
+| `--dual`       | `OFF`, `DE_ENPY`, `DE_EINT`                                | `OFF`   | Enable dual energy formalism | Not supported for `--flu_scheme=RTVD`. `DE_EINT` is not supported yet. | <a name="--dual"></a> `DUAL_ENERGY` |
+| `--mhd`        | `true`, `false`                                            | `false` | Magnetohydrodynamics | - | <a name="--mhd"></a> `MHD` |
+| `--srhd`       | `true`, `false`                                            | `false` | Special relativistic hydrodynamics | Must adopt `--eos=TAUBMATHEWS` | <a name="--srhd"></a> `SRHD` |
+| `--cosmic_ray` | `true`, `false`                                            | `false` | Cosmic rays | Must adopt `--eos=COSMIC_RAY` | <a name="--cosmic_ray"></a> `COSMIC_RAY` |
+| `--eos`        | `GAMMA`, `ISOTHERMAL`, `COSMIC_RAY`, `TAUBMATHEWS`, `USER` | Depend  | [[Equation of state \| equation-of-state]] | The following options only support `GAMMA`: `--flu_scheme=RTVD/CTU`, `--flux=EXACT/ROE`, `--comoving`, `--dual`; see also [[--barotropic \| Installation:-Option-List#--barotropic]] | <a name="--eos"></a> `EOS` |
+| `--barotropic` | `true`, `false`                                            | Depend  | Is [[--eos \| Installation:-Option-List#--eos]] barotropic? | Must be disabled for `--eos=GAMMA` and enabled for `--eos=ISOTHERMAL` | <a name="--barotropic"></a> `BAROTROPIC_EOS` |
+
+## ELBDM Options
+-- see [[ELBDM]] for the related runtime parameters and other settings
```

Comment:
Add "; must enable --model=ELBDM" at the end of this sentence. Then you can remove "Must set `--model=ELBDM`" from the `Restriction` column.